com.unity.render-pipelines.high-definition/Editor/RenderPipeline/Settings/QualitySettingsPanel.cs

```diff
@@ -70,6 +72,10 @@ public void OnActivate(string searchContext, VisualElement rootElement)
             {
                 m_HDRPAssets.Clear();
                 PopulateHDRPAssetsFromQualitySettings(m_HDRPAssets);
+
+                var currentAsset = QualitySettings.GetRenderPipelineAssetAt(QualitySettings.GetQualityLevel());
```

Comment:
`QualitySettings.GetRenderPipelineAssetAt(QualitySettings.GetQualityLevel());` is equivalent to `QualitySettings.currentPipeline;`
However it may not be the one in use if null.